change/@fluentui-react-charting-4c197bf1-3dca-480d-aa17-d97a170246ff.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv8 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33479&commitId=62884cd4bf7745063353603a3d8d1a04de952ea1&env=prod&clientType=FLUENTUI)


  <details><summary><b>Callout</b> 7 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Callout.Right top edge.chromium.png|1127| Changed|
|Callout.Left top edge.chromium.png|1949| Changed|
|Callout.Root.chromium.png|2309| Changed|
|Callout.Left center.chromium.png|2616| Changed|
|Callout.No beak.chromium.png|2306| Changed|
|Callout.Bottom Left Edge RTL.chromium.png|2178| Changed|
|Callout.Gap space 25.chromium.png|2309| Changed|

</details>


  <details><summary><b>Coachmark</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Coachmark.Collapsed.chromium.png|159| Changed|

</details>


  <details><summary><b>react-charting-AreaChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-AreaChart.Custom Accessibility.chromium.png|11| Changed|

</details>


  <details><summary><b>react-charting-LineChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-LineChart.Gaps.chromium.png|1| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>